Demo_ScanAndProcess_3D.m

```diff
@@ -48,10 +48,20 @@
     warning('Because we use gel above tissue to find focus position. It is important to have at least one of the z-stacks in the gel. Consider having the minimum zToScan_mm to be -100e-3[mm]')
 end
 
+% Quick pre-scan to identify tissue surface and verify it is at OCT focus
+if ~skipScanning
+    [surfacePosition_mm, x_mm, y_mm, isSurfaceFocused] = yOCTScanAndFindTissueSurface(... 
+            'xRange_mm', xOverall_mm,...
+            'yRange_mm', yOverall_mm,...
+            'pixel_size_um', 10,...
+            'focusPositionInImageZpix', focusPositionInImageZpix,...
+            'dispersionQuadraticTerm', dispersionQuadraticTerm);
+    assert(isSurfaceFocused, 'Adjust the stage as needed and rerun the script.');
+    disp('Continuing...');

```

Comment:
See below the rest of the script uses "fprintf('%s Scanning Volume\n',datestr(datetime));" instead of disp.
Can you use the same to keep formatting uniform, place the message before the function is run